Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.yml

```diff
@@ -1,49 +1,72 @@
 category: IT Services
+sectionOrder:
+- Connect
+- Collect
 commonfields:
   id: Microsoft Graph Device Management
   version: -1
 configuration:
-- defaultvalue: https://graph.microsoft.com
```

Comment:
removing required field is always a bad idea - what happens if a customer had an instance already? It means we are breaking BC.

Reply:
Of course.
The field still exists, it just moved it down to the end of the params, as we rather users should not deal with it.
No BC to worry about.